packages/mason_logger/lib/src/log_level.dart

```diff
@@ -0,0 +1,28 @@
+/// Indicates the desired logging level.
+enum LogLevel {
+  /// Logs everything, no matter what levels will be added in the future.
+  ///
+  /// This is the default level.
+  all,
```

Comment:
Shouldn't all be a static property that directs to values[0]? Feels like `LogLevel.values` shouldn't include all (and we should avoid allowing people to set it in some places)

```
static LogLevel get all => values[0];
```